uaclient/system.py

```diff
@@ -431,8 +431,11 @@ def _subp(
     bytes_args = [
         x if isinstance(x, bytes) else x.encode("utf-8") for x in args
     ]
-    if env:
-        env.update(os.environ)
```

Comment:
I think the bug is that `{}` is falsey, so it currently won't get the os env.

So an alternative fix would be to replace this line with:
```python
if env is not None:
```